libraries/rush-lib/src/utilities/Utilities.ts

```diff
@@ -773,39 +785,52 @@ export class Utilities {
 
     const escapedArgs: string[] = args.map((x) => Utilities.escapeShellParameter(x));
 
-    let result: child_process.SpawnSyncReturns<string | Buffer> = child_process.spawnSync(
-      escapedCommand,
-      escapedArgs,
-      options
-    );
+    const process: child_process.ChildProcess = child_process.spawn(escapedCommand, escapedArgs, options);
 
-    // eslint-disable-next-line @typescript-eslint/no-explicit-any
-    if (result.error && (result.error as any).errno === 'ENOENT') {
-      // This is a workaround for GitHub issue #25330
-      // https://github.com/nodejs/node-v0.x-archive/issues/25330
-      //
-      // TODO: The fully worked out solution for this problem is now provided by the "Executable" API
-      // from @rushstack/node-core-library
-      result = child_process.spawnSync(command + '.cmd', args, options);
+    const stdoutBuffers: string[] = [];
+    process.stdout?.on('data', (chunk) => {
+      stdoutBuffers.push(chunk.toString());
+    });
+
+    const stderrBuffers: string[] = [];
+    process.stderr?.on('data', (chunk) => {
+      stderrBuffers.push(chunk.toString());
+    });
+
+    let error: Error | undefined;
+    try {
+      await once(process, 'close');
```

Comment:
There was some outcome that we came to with which event we should wait on before exiting. Is this the one we settled on?

Reply:
Updated to use `Executable.waitForExitAsync`